### PR TITLE
Update ESLint Config #407

### PR DESCRIPTION
**Pull request by Mottoweb**
Hey,

I stumbled upon #407, so i thought i could give it a go for a first good PR.

Let me know if i missed anything.
Things were done:
1. Remove anything lint related from package.json
2. Create .eslintrc.json with same rules.
3. Remove custom rules.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (kwelch) [inline .eslintrc.json]: I believe we still want the config to live in `package.json`. This was done intentionally to reduce errors when the project was copied. See #405

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "root": true,
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
```
Comment (kwelch) [inline .eslintrc.json]: I would expect the import plugin to be listed in plugs. Is it inherited by another?

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "plugins": [
+    "react"
```
Comment (kwelch) [inline .eslintrc.json]: I like that we are importing this, I do believe we should find a export preset of rules we can extend here to cover (most or all) of the rules we had previously set.

**Comment by kwelch** [comment]
Thanks, I have put in a few comments. I appreciate you taking a stab at this. :+1:

File: .eslintrc.json (hunk 3 above)
Comment (Mottoweb) [reply .eslintrc.json]: Then what is the point of removing custom rules and creating dependency on other packages? The very last comment section of the issue #407 suggests a PR that removes custom rules, imports default, or am i wrong?

File: .eslintrc.json (hunk 1 above)
Comment (Mottoweb) [reply .eslintrc.json]: Ok no prob)

File: .eslintrc.json (hunk 2 above)
Comment (Mottoweb) [reply .eslintrc.json]: Could not trace where exactly it is inherited from but, it is.

**Comment by kwelch** [comment]
@coryhouse & @nickytonline What do you think? I still want to pull it down and run lint a few times and fix to see if it causes any changes.

**Comment by nickytonline** [comment]
I'm OK with you testing it locally to see if any other changes are required. It's using recommended/defaults for linting, so unless you see something glaring after you test it, I'd say we're probably good to go if Cory's OK with the changes.

**Comment by coryhouse** [comment]
Linting runs clean for me so PR looks 👍

**Comment by nickytonline** [comment]
Thanks for the PR @Mottoweb!

**Comment by Mottoweb** [comment]
Glad to be of help!

**Pull request merged**